Mackbellemore/soen-390-team07
# feat: Log API calls to logdna in production only

To test this PR:

1. Log into [logdna](https://app.logdna.com/account/signin) with credentials (see discord for secrets)
2. Create .env file in server folder
3. Add environment variable in .env file with key=value as follows: LOGDNA= Logdna ingestion key (see discord secrets)
4. Make an API call 
5. Nothing should appear in Logdna

To log an api call to logdna, do the following:

1. Go to default.js file in server folder
2. Change line 12 `zeetEnv: process.env.ZEET_ENVIRONMENT || 'local',` to `zeetEnv: process.env.ZEET_ENVIRONMENT || 'develop',`
